### DISCUSSION TITLE
CLAP plugin versions?

### DISCUSSION THREAD

**Discussion by 38github** [GuitarML/PedalNetRT#36] [Ideas]
I tried out your EpochAmp and it is really interesting. As soon as I get in the mood to play guitar for some hours I will really test it out. The machine learning really seem to make a great difference to amp simulation.

Have you thought of making CLAP versions? I noticed in Windows under Reaper that plugins that had both VST3 and CLAP version showed an interesting lower CPU utilization when using the CLAP versions.

**Comment by GuitarML** [comment]
@38github At the moment I have no plans to build a CLAP version of EpochAmp. CLAP is something I haven't looked into yet, but it would be nice to be able to support it, especially if it's a more efficient format. Thanks for the suggestion!
